makefile

```diff
@@ -0,0 +1,62 @@
+SERVICES = rust-actix
+COMPOSE_FILE_STANDARD = docker-compose.yml
+TEST_COMMAND = pytest tests/test_api_endpoints.py
+LOCAL_DATABASE = ./database/docker-compose.yml
+DATABASE_TABLES = users, followers, messages, latest
+DELAY_TEST_EXECUTION_SECONDS = 3
+
+.PHONY: start-local-db
+start-local-db:
+	@echo "Starting the database container..."
+	docker-compose -f $(LOCAL_DATABASE) up -d
+
+.PHONY: stop-local-db
+stop-local-db:
+	@echo "Stopping and removing the database container..."
+	@docker-compose -f $(LOCAL_DATABASE) stop
+	@docker-compose -f $(LOCAL_DATABASE) rm -f database
+
+.PHONY: clean-db
+clean-db:
+	@echo "Cleaning the database..."
+	@export PGPASSWORD=pass
+	@docker-compose -f $(LOCAL_DATABASE) exec local_database psql -U user -d waect -c "TRUNCATE TABLE $(DATABASE_TABLES) ;"
```

Comment:
"local_database" is must "database" now, or am i wrong?

Reply:
yes